docs/modules/ROOT/pages/getting-started/configuration.adoc

```diff
@@ -0,0 +1,257 @@
+metadata:
+  name: kubearchive
+  namespace: default
+spec:
+  resources:
+    - selector:
+        apiVersion: v1
+        kind: Pod
+      archiveWhen: status.phase == "Succeeded"
+----
+
+To see it in action apply the KubeArchiveConfig in your namespace and
+run a pod:
+
+[source,bash]
+----
+kubectl run fedora --image quay.io/fedora/fedora:latest --restart Never -- sleep 10
+----
+
+The pod will not appear in KubeArchive until it completes. After completion, it is present in KubeArchive:
+
+[source,bash]
+----
+$ curl --insecure \
+    -H "Authorization: Bearer ${SA_TOKEN}" \
+    https://localhost:8081/api/v1/namespaces/default/pods \
+    | jq -r '.items[] | [.metadata.name, .metadata.uid] | @csv'
+
+"fedora", "a3bdb6d2-b683-4913-9d24-e01af60c94e3"
+----
+
+[NOTE]
+====
+We use `jq` in these examples to reduce output.
+====
+
+The pod remains in Kubernetes, occupying space:
+
+[source,bash]
+----
+$ kubectl get pods --namespace default
+NAME     READY   STATUS      RESTARTS   AGE
+fedora   0/1     Completed   0          4m50s
+----
+
+== `deleteWhen`: Deleting Resources
+
+They key feature of KubeArchive is deleting resources. This feature keeps
+the Kubernetes cluster free of resources that are not needed anymore. To enable
+deletion of resources use the `deleteWhen` key.
+
+[IMPORTANT]
+====
+When resources are deleted with `deleteWhen` they get archived too.
+====
+
+The following KubeArchiveConfig configures KubeArchive to delete (and archive)
+pods when they match `status.phase == "Succeeded"`:
+
+[source,yaml]
+----
+---
+apiVersion: kubearchive.kubearchive.org/v1alpha1
+kind: KubeArchiveConfig
+metadata:
+  name: kubearchive
+  namespace: default
+spec:
+  resources:
+    - selector:
+        apiVersion: v1
+        kind: Pod
+      deleteWhen: status.phase == "Succeeded"
+----
+
+To see it in action apply the KubeArchiveConfig in your namespace and
+run a pod:
+
+[source,bash]
+----
+kubectl run auto-deleted --image quay.io/fedora/fedora:latest --restart Never -- echo "sleep 10"
+----
+
+Start watching pods to see how this pod is created, executed, completed and deleted:
+
+[source,bash]
+----
+$ kubectl get pods -w
+NAME        READY   STATUS              RESTARTS   AGE
+auto-deleted   0/1     ContainerCreating   0          2s
+auto-deleted   1/1     Running             0          2s
+auto-deleted   0/1     Completed           0          13s
+auto-deleted   0/1     Completed           0          14s
+auto-deleted   0/1     Terminating         0          14s
+auto-deleted   0/1     Terminating         0          14s
+----
+
+After the pod has been deleted from the cluster by KubeArchive, retrieve it:
+
+[source,bash]
+----
+$ curl --insecure \
+    -H "Authorization: Bearer ${SA_TOKEN}" \
+    https://localhost:8081/api/v1/namespaces/default/pods \
+    | jq -r '.items[] | [.metadata.name, .metadata.uid] | @csv'
+
+...
+"auto-deleted","64c48176-ba8c-4f2a-a662-1fd660f7a3b6"
+----
+
+== `archiveOnDelete`: Archiving on Deletion
```

Comment:
Based o n the previous suggestion:

```suggestion
== `archiveOnDelete`: Archiving on Deletion from the cluster
```

I kept the "delete" verb in here to match the `archiveOnDelete` key but added the context of "from the cluster" to be more explicit of the scope of that deletion.